RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateReco.h

```diff
@@ -50,49 +53,52 @@
 #ifndef SiPixelTemplateReco_h
 #define SiPixelTemplateReco_h 1
 
-#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateDefs.h"
-
 #ifndef SI_PIXEL_TEMPLATE_STANDALONE
+#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateDefs.h"
 #include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplate.h"
 #else
+#include "SiPixelTemplateDefs.h"
 #include "SiPixelTemplate.h"
 #endif
 
 #define N2D 500
 
 #include <vector>
 
+#ifndef SiPixelTemplateClusMatrix
+#define SiPixelTemplateClusMatrix 1
```

Comment:
Sorry for the delay.  Morris and I looked at all of these, so here are responses one at the time.

This one is used to block duplicate inclusion the definition of ClusMatrix object.  (Why it wasn't called SiPixelTemplateClusMatrix, I don't know.)  This was added by Vincenzo I. some time ago.